lib/pbench/cli/agent/commands/tools/list.py

```diff
@@ -29,16 +29,19 @@ def print_results(toolinfo: dict, with_option: bool) -> bool:
         """
         printed = False
         for group, gval in sorted(toolinfo.items()):
-            for host, tools in sorted(gval.items()):
+            for host, hostitems in sorted(gval.items()):
+                label = hostitems["label"]
+                host_string = f"host: {host}" + (f", label: {label}" if label else "")
+                tools = hostitems["tools"]
                 if tools:
                     if not with_option:
-                        s = ", ".join(sorted(tools.keys()))
+                        tool_string = ", ".join(sorted(tools.keys()))
                     else:
-                        tools_with_options = [
+                        tools_with_options = (
```

Comment:
Not a complaint or objection ... just a casual observation that a shorter name should convince `black` to keep this on a single line. 😆

Reply:
I actually like the formatting here: it attracts the reader's attention to the special role that the parens play. That gets lost if everything is squished into one line.